hack/test-integration.sh

```diff
@@ -28,7 +28,9 @@ tags="${OS_TEST_TAGS:-integration no-docker}"
 
 export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
 TMPDIR=${TMPDIR:-/tmp}
-export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"}
+export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"} 
+
+#"
```

Comment:
My bash-fu must be weak.  Does this do anything?  It's just a weird comment line, right?